smart-contracts/contracts/interfaces/IUnlock.sol

```diff
@@ -6,9 +6,22 @@ pragma solidity 0.5.16;
  * @author Nick Furfaro (unlock-protocol.com)
 **/
 
-interface IUnlock {
+interface IUnlock
+{
+  /**
+   * The struct for a lock
+   * We use deployed to keep track of deployments.
+   * This is required because both totalSales and yieldedDiscountTokens are 0 when initialized,
+   * which would be the same values when the lock is not set.
+   */
+  struct LockBalances
+  {
+    bool deployed;
+    uint totalSales; // This is in wei
+    uint yieldedDiscountTokens;
+  }
```

Comment:
Not sure about this, moving storage to the interface?

Reply:
It's just the struct itself - not any data.  We will need this in the interface once we switch to abiv2 or Solidity 0.6 and want to start exposing structures publicly.

Reply:
copy that.

Reply:
But for now we could move it back...

Reply:
Yea I'm going to move it back for now. We can revisit when we need it, but for now this is the first instance of info only available in the interface...

Reply:
Allright. I'm fine either way here if we'll need it soon enough.